public/app/percona/dbaas/components/DBaasRouting/DBaaSRouting.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC, useEffect, useMemo } from 'react';
+import { getKubernetes as getKubernetesSelector } from '../../../shared/core/selectors';
+import { fetchKubernetesAction } from '../../../shared/core/reducers';
+import { CHECK_OPERATOR_UPDATE_CANCEL_TOKEN, GET_KUBERNETES_CANCEL_TOKEN } from '../Kubernetes/Kubernetes.constants';
+import { useCancelToken } from '../../../shared/components/hooks/cancelToken.hook';
+import { useDispatch, useSelector } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import { Spinner, useStyles } from '@grafana/ui/src';
+import { getStyles } from '../DBaasRouting/DBaasRouting.styles';
+
+export const DBaaSRouting: FC<{}> = ({}) => {
```

Comment:
This could be probably simplified, as there are no props used the `{}` can be removed.